docs/multicluster/user-guide.md

```diff
@@ -559,6 +559,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Antrea Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster allows traffic between Pods in different clusters. If you want to enable this feature,
```

Comment:
Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters through the Multi-cluster Gateways. Pod IPs can be reached in all member clusters within a ClusterSet.